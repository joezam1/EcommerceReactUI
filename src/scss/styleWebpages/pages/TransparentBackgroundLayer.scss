*{
    box-sizing:border-box;
  }
  
  .transparent-layer{
    background-color:black;
    display:none;
    left:0;
    opacity:0.75;
    position:absolute;
    top:0;
    z-index:5000;
  }
  