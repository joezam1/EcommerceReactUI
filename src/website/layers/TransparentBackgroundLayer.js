import React from 'react';

export default function TransparentBackgroundLayer(){

    return( <div className="transparent-brackground-layer">
        <div id="transparent-layer-id" className="transparent-layer"></div>
      </div>)
}