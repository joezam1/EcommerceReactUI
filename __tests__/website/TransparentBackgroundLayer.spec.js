import React from 'react';

import {render, cleanup, waitFor, fireEvent} from '@testing-library/react';

import TransparentBackgroundLayer from '../../src/website/layers/TransparentBackgroundLayer.js';

xdescribe('File TransparentBackgroundLayer', function(){
    afterEach(cleanup);
    test('create a snapshot', function(){
        let container = render(<TransparentBackgroundLayer/>);
        expect(container.asFragment(<TransparentBackgroundLayer/>)).toMatchSnapshot();

    })
})