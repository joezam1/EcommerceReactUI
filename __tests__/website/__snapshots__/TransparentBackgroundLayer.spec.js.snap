// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`File TransparentBackgroundLayer create a snapshot 1`] = `
<DocumentFragment>
  <div
    class="transparent-brackground-layer"
  >
    <div
      class="transparent-layer"
      id="transparent-layer-id"
    />
  </div>
</DocumentFragment>
`;
